core/common/src/main/java/alluxio/exception/PreconditionMessage.java

```diff
@@ -74,29 +70,14 @@
   // SEMICOLON! minimize merge conflicts by putting it on its own line
   ;
 
-  private final MessageFormat mMessage;
+  private final String mMessage;
 
   PreconditionMessage(String message) {
```

Comment:
This is out of scope for this PR, but do we get anything out of this being an enum as opposed to a bunch of string constants? With constants we wouldn't even need to call `toString()`